The Polish dictionary and language package

oskar-gmerek opened the pull request.

MrWook commented on packages/languages/pl/package.json:
```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@zxcvbn-ts/language-en",
```
@zxcvbn-ts/language-pl

MrWook commented on packages/languages/pl/README.md:
```diff
@@ -0,0 +1,46 @@
+## Sources
+
+1. First and last names data use to be a dictionary for zxcvbn-ts are extracted from files based on Polish Government Public Datasets (Public Domain) and available publicly on <https://github.com/oskar-gmerek/polish_database> (MIT licensed)
+  [*] Data is based on Last and First names registered under Polish Universal Electronic System for Population Register and may contains also names of foreigners registered under this Register, (ex. refugees from Ukraine who were given a PESEL number after the Russian Invasion of Ukraine)
```
If you got the data from a public dataset. It would be nice if you use it directly and if necessary create an generator for the file like the other language packages if the data was available in the internet to download.
If this is not possible because its behind a login or whatever it would be nice if you write where you got the data from

MrWook reviewed (commented):
In you name lists some names contain a lot of whitespace.

oskar-gmerek commented:
@MrWook 

I removed undesirable whitespaces and corrected package name.

About the source. I got this data from a very large file I was used some time ago. I can't find it again. I'm sure that was from some of government's website, but I'm not sure from which one.

If there will be interest then I can do some better research and update this package with a more documented source. But at the moment I do not have enough time for proper research.
